### PR TITLE
feat: add Safe{Pass} button to the navigation

### PR DESCRIPTION
**Pull request by DiogoSoaress**
## Figma
https://www.figma.com/file/qtXXlwdslAze6yNbLLzIRa/Website-redesign-2.0?type=design&node-id=12520-146768&mode=design&t=SrG7ZuhjV8XVWZQK-0

### CODE REVIEW

**Comment by TanyaEfremova** [comment]
All good!

File: src/components/common/Header/NavigationButton.tsx [hunk 1]
```diff
@@ -17,20 +17,22 @@ const NavigationButton = ({ category, items, subMenuOpen, onItemClick }: Navigat
 
   const isOpen = subMenuOpen === category
 
+  const categoryString = typeof category === 'string' ? category : 'unknown category'
```
Comment (iamacook) [inline src/components/common/Header/NavigationButton.tsx:20]: All `catgory`s are capitalised words. Do we wants to use `"Unknown"` instead?

File: src/components/common/Header/index.tsx [hunk 2]
```diff
@@ -46,40 +46,38 @@ const Header = () => {
...
-                    items={items ?? []}
-                    subMenuOpen={subMenuOpen}
-                    onItemClick={closeMobileNavigation}
-                  />
-                )}
-              </li>
-            </>
+          {navCategories.map(({ category, items, href }) => (
+            <li
+              key={typeof category === 'string' ? category : href}
```
Comment (iamacook) [inline src/components/common/Header/index.tsx:51]: Nit: I'd suggest keeping the key simple:

```suggestion
              key={href}
```

File: src/components/common/Header/index.tsx [hunk 3]
```diff
@@ -46,40 +46,38 @@ const Header = () => {
...
+              onMouseEnter={toggleCategoryOpen(typeof category === 'string' ? category : null)}
+              onFocus={toggleCategoryOpen(typeof category === 'string' ? category : null)}
+              onMouseLeave={toggleCategoryOpen(null)}
+              onBlur={toggleCategoryOpen(null)}
```
Comment (iamacook) [inline src/components/common/Header/index.tsx:55]: You can abstract these to two variables before rendering, e.g.:

```ts
const onOpen = toggleCategoryOpen(typeof category === 'string' ? category : null)
const onClose = toggleCategoryOpen(null)
```

Comment (DiogoSoaress) [reply src/components/common/Header/index.tsx]: I've undone these changes. Included `JSX.Element` as a possible input type instead and added an early return for it.

**Pull request merged**